packages/logger/src/keys.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/logger
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export namespace Logger {
+  export namespace SequenceActions {
+    export const SETUP_LOGGER  = 'logger.sequenceActions.setupLogger';
+  }
+  export namespace System {
+    export const LOGGER_PROVIDER  = 'logger.system.loggerProvider';
```

Comment:
The reason I tentatively used system.logger is that I was thinking there might be two or more loggers simultaneously and this one is so called `default logger`, but it was long time ago and my understanding/assumption at that time was less mature.
```
export namespace Logger {
  export const SIMPLE_LOGGER  = 'logger.simpleLogger';
```